CHANGELOG.md

```diff
@@ -1,4 +1,23 @@
 # Changelog
+## 1.2.0
+### Added
+
+*no additions*
+
+### Changed
+
+*no changes*
```

Comment:
When a section is empty we can probably just leave it out altogether. Up to you.